packages/server/src/fhir/search.ts

```diff
@@ -733,27 +739,38 @@ function buildFilterParameterComparison(
 
 /**
  * Adds a string search filter as "WHERE" clause to the query builder.
+ * @param table - The table in which to search.
  * @param details - The search parameter details.
  * @param operator - The search operator.
  * @param values - The string values to search against.
  * @returns The select query condition.
  */
-function buildStringSearchFilter(details: SearchParameterDetails, operator: Operator, values: string[]): Expression {
+function buildStringSearchFilter(
+  table: string,
+  details: SearchParameterDetails,
+  operator: Operator,
+  values: string[]
+): Expression {
+  const column = new Column(details.array ? undefined : table, details.columnName);
```

Comment:
`undefined`?  Curious about why that was necessary.

Reply:
This is basically a workaround carried forward from the existing implementation, which requires it to deal with the `ArraySubquery` where the inner subquery needs to refer to the unqualified column name (aliased from `unnest()`)